### PR TITLE
Command Line Basics: Update assignment resource

### PR DESCRIPTION
**Pull request by xandora**
Change assignment from using Conquering The Command Line to something more exercise based by Software Carpentry Foundation

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

The Conquering the Command Line resource is a good overview, but it tells rather than teaches. Software Carpentry Foundation lessons are designed for total novices and are a good exercise based entry point into learning the CLI.

**2. This PR:**

* Replaces Conquering the Command Line as the assignment task with a similar task created the Software Carpentry Foundation.

**3. Additional Information:**

### CODE REVIEW

File: foundations/installations/command_line_basics.md [hunk 1]
```diff
@@ -35,13 +35,21 @@ This section contains a general overview of topics that you will learn in this l
 * Use the command line to rename or destroy a directory and a file.
 * Use the command line to open a file or folder in a program.
 
-### Assignment
+### NEW Assignment
```
Comment (xandora) [inline foundations/installations/command_line_basics.md:38]:
```suggestion
### Assignment
```

**Comment by xandora** [comment]
Good catch, we definitely don't need anyone installing R. Will amend.

File: foundations/installations/command_line_basics.md [hunk 2]
```diff
@@ -40,8 +40,16 @@ This section contains a general overview of topics that you will learn in this l
...
-  2. Read through [chapter 1 of Conquering the Command Line](https://www.softcover.io/read/fc6c09de/unix_commands/basics).
+1. Visit [The Unix Shell](https://swcarpentry.github.io/shell-novice/) course designed by the Software Carpentry Foundation. There you will find a full complement of lessons on using the CLI, but for now just focus on completing the following lessons:
+    
+    - Setup (this step is required to complete some of the exercises within the following lessons)
```
Comment (xandora) [inline foundations/installations/command_line_basics.md:45]:
```suggestion
    - Setup (Download the exercise files, but skip the Software Install step, you already have everything you need)
```

**Comment by xandora** [comment]
> @xandora Did you want another set of eyes on this PR? Or are you good with merging.

Oh, well I'm happy merging it as is. I didn't notice you had actually responded. Must've missed the email.

**Comment by codyloyd** [comment]
i like this 👍🏽  

good work @xandora

**Pull request merged**